stravalib/model.py

```diff
@@ -856,11 +1155,12 @@ class BaseActivityZone(
     A collection of :class:`stravalib.model.DistributionBucket` objects.
     """
 
-    # field overrides from superclass for type extensions:
-    distribution_buckets: Optional[List[DistributionBucket]] = None
+    # Field overrides from superclass for type extensions:
+    # Using type that is currently mimicking legacy behavior...
+    distribution_buckets: Optional[List[DistributionBucket]] = None  # type: ignore
 
-    # overriding the superclass type: it should also support pace as value
-    type: Optional[Literal["heartrate", "power", "pace"]] = None
+    # TODO: ignoring type given legacy support will be deprecated
+    type: Optional[Literal["heartrate", "power", "pace"]] = None  # type: ignore
```

Comment:
More precise ignore

Reply:
cool - i'll do a find and replace for all ignores with out a specific error specified!